lib/utils.ts

```diff
@@ -87,3 +94,14 @@ export const verifyClaim = (token: string, { iss }: ITokenPayload) => {
   const addressFromDigest = recoverAddress(digest, signature)
   return decodedAddress === addressFromDigest ? iss : ''
 }
+
+//used to compare ids in DID Since the referenced pubkey id differs slighty from the auth reference Id
+export const areLinked = (authId :string, pubKeyID: string)  =>  {
```

Comment:
Just curious, why put this in `utils.ts`?

Reply:
I used this pattern thinking that this function could be needed in other classes. But yes, I agree with your views about "utils"